op-node/rollup/derive/attributes.go

```diff
@@ -124,6 +124,14 @@ func (ba *FetchingAttributesBuilder) PreparePayloadAttributes(ctx context.Contex
 		upgradeTxs = append(upgradeTxs, fjord...)
 	}
 
+	if ba.rollupCfg.IsIsthmusActivationBlock(nextL2Time) {
+		isthmus, err := IsthmusNetworkUpgradeTransactions()
```

Comment:
This is going to conflict https://github.com/ethereum-optimism/optimism/pull/14110 depending on which is merged first